plugins/woocommerce/client/blocks/assets/js/blocks/product-description/block.json

```diff
@@ -8,6 +8,10 @@
 	"category": "woocommerce",
 	"textdomain": "woocommerce",
 	"usesContext": [ "postId", "postType", "queryId" ],
+	"ancestor": [
+		"woocommerce/single-product",
+		"woocommerce/product-template"
```

Comment:
We should also add `core/post-template` to this list, as otherwise it prevents users from using this in a query loop based on products:
<img width="674" alt="Screenshot 2025-03-12 at 4 14 39 PM" src="https://github.com/user-attachments/assets/e2877ddb-5158-4e4d-8b58-9eb1d3b8249a" />

Reply:
Sure, I can make this change. But the catch is that after adding this new property we will need some additional logic. So here is a new commit 380bd8e with all the changes. Let me know if that looks good to you.